clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/model/HealthState.groovy

```diff
@@ -2,5 +2,5 @@
 package com.netflix.spinnaker.clouddriver.model
 
 enum HealthState {
-  Up, Down, Unknown, Starting, OutOfService
+  Up, Down, Unknown, Starting, OutOfService, Succeeded, Failed
```

Comment:
Feels like we're trying to shoehorn some sort of result into the health state. Is there a need to keep the container around if it is no longer executing?